docs/docsite/rst/dev_guidelines.rst

```diff
@@ -480,20 +480,135 @@ When you make a call using boto3, you will probably get back some useful informa
 should return in the module.  As well as information related to the call itself, you will also have
 some response metadata.  It is OK to return this to the user as well as they may find it useful.
 
-Boto3 returns all values CamelCased.  Ansible follows Python standards for variable names and uses
-snake_case. There is a helper function in module_utils/ec2.py called ``camel_dict_to_snake_dict``
+Boto3 commonly returns all keys CamelCased.  Ansible follows Python standards for variable names and uses
+snake_case.  There is a commonly used helper function ``ansible.module_utils.common.dict_transformations.camel_dict_to_snake_dict``
```

Comment:
snake_case. There is a used case in module_utils/ec2.py called ``camel_dict_to_snake_dict`` it is a helper function.

Reply:
@Dule-mart updated with suggestions and modified a little. Let me know if there's anything else to clear up!